apisix/core/config_etcd.lua

```diff
@@ -223,10 +241,21 @@ local function do_run_watch(premature)
             break
         end
 
-        if res.result.created then
-            goto watch_event
-        end
-
+        --[[
```

Comment:
found this bug by https://github.com/apache/apisix/blob/master/t/cli/test_etcd_sync_event_handle.sh